test/opexecuter.cpp

```diff
@@ -320,8 +321,17 @@ void OpExecuter::ExecuteOnNGraph(vector<Tensor>& ngraph_outputs) {
   }
 
   // Create nGraph backend
-  // Create the nGraph backend
-  auto backend = ng::runtime::Backend::create("CPU");
+  // If NGRAPH_TF_BACKEND is set create that backend
+  // Else create backend of type ng_backend_name
+  const char* ng_backend_env_value = std::getenv("NGRAPH_TF_BACKEND");
+  if (ng_backend_env_value != nullptr) {
+    string backend_env = std::string(ng_backend_env_value);
+    if (!backend_env.empty()) {
+      ng_backend_name = backend_env;
+    }
+  }
+  NGRAPH_VLOG(5) << " Creating NG Backend " << ng_backend_name;
+  auto backend = ng::runtime::Backend::create(ng_backend_name);
 
```

Comment:
and in case we do not apply the check i mention above, we can try-catch the backend creation, to know that incorrect backends were not requested